changelog/release-support-lifecycle.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: RisingWave release and support lifecycle
+sidebarTitle: Release and support lifecycle
+description: "This document provides an overview of the RisingWave versioning scheme, release cadence, and a summary of the support lifecycle for different versions. It also links to detailed policies for both version and feature deprecation."
+---
+
+This page provides a high-level overview of how RisingWave versions are numbered, how frequently releases occur, and how long different versions are supported. It serves as a central point for understanding the overall release and support process.
+
+## Versioning scheme (semantic versioning)
+
+RisingWave adheres to the [Semantic Versioning (SemVer)](https://semver.org/) standard, using a three-part numbering system: `Major.Minor.Patch`.
+
+*   **Major version (e.g., 2.0.0):** Represents significant releases with substantial new features, architectural changes, and potentially breaking changes that might require migration.
+*   **Minor version (e.g., 2.1.0):** Represents regular releases with new features, enhancements, and bug fixes. Minor versions are generally backward compatible within the same major version.
+*   **Patch version (e.g., 2.1.1):** Represents releases focused solely on bug fixes and security patches. Patch versions do not introduce new features or breaking changes.
```

Comment:
This description can cause misunderstanding when coping with 1.10, 2.0 and 2.1. The difference between 2.0 and 2.1 is more significant than between 1.10 and 2.0. User need to do migration between 2.0 and 2.1, but don't need to do it between 1.10 and 2.0.

Reply:
This section is intended to establish the expectation. We have regular releases (45 days or so) for 2.x versions, so it's accurate to refer to these as regular releases. 2.1 is more of an exception in the sense that it contains significant and breaking changes but we didn't increment the major version. If we describe how we manage versions previously and use that as the policy / guideline, it would be something like: Sometimes we increment versions only for marketing purposes, sometimes a minor version can contain significant changes that need additional migration efforts. It would appear to users that we have poor planning and version management.

Reply:
Let me add a note saying this is the typical scenario, and in the past, we didn't strictly followed the practice but going forward will adhere to the semver standard.